hooks/useForm.tsx

```diff
@@ -24,7 +24,20 @@ const useForm = <T extends object>(initialValues: T) => {
 
   const checkFormFilled = useCallback(() => {
     const keys = Object.keys(values) as (keyof T)[];
-    const emptyValues = keys.filter((name) => values[name] === '');
+    const emptyValues = keys.filter((name) => {
+      const value = values[name];
+
+      switch (typeof value) {
+        case 'object':
+          return !(values[name] as File[]).length;
+        case 'string':
+          return value === '';
+        case 'number':
+          return !value;
+        default:
+          return !value;
```

Comment:
case 'number' 제거해줘도 괜찮지 않을까요?

```suggestion
        default:
          return !value;
```

Reply:
`number` 쪽에서 다른 처리를 해줘야 할 수도 있을 것 같아서 임의로 넣어놨습니다.